File: Treinamento/Java/Estrutura_sequencial/Questão11.java
import java.util.Scanner;
public class Questão11 {
    public static void main(String[] args) {
        Scanner scan;
        scan = new Scanner(System.in);

        System.out.print("Digite o ganho horário: ");
        double g = scan.nextDouble();

        System.out.print("Digite a jornada horária: ");
        int j = scan.nextInt();

        double s = g * j * 20;

        double ir = s * 0.11;
        double inss = s * 0.08;
        double sindic = s * 0.05;

        System.out.printf("Salário bruto: R$%.2f reais\n", s);
        System.out.printf("Imposto de renda: R$%.2f reais\n", ir);
        System.out.printf("INSS: R$%.2f reais\n", inss);
        System.out.printf("Sindicato: R$%.2f\n", sindic);
        System.out.printf("Salário líquido: R$%.2f reais\n", (s - ir - inss - sindic));
    }
}

File: Treinamento/C/Estrutura_sequencial/Questão11.c
#include <stdio.h>

int main() {
    double ganho_horario;
    int hora_dia;

    printf("Digite o seu ganho horário: ");
    scanf("%lf", &ganho_horario);

    printf("Digite a sua jornada diária: ");
    scanf("%i", &hora_dia);

    double s_bruto = ganho_horario * hora_dia * 20;
    double ir = s_bruto * 0.11;
    double inss = s_bruto * 0.08;
    double sindicato = s_bruto * 0.05;

    printf("Salário bruto: R$%lf reais\n", s_bruto);
    printf("Imposto de renda: R$%lf reais\n", ir); 
    printf("INSS: R$%lf reais\n", inss); 
    printf("sindicato R$%lf reais\n", sindicato); 
    printf("Salário líquido: R$%lf reais\n", s_bruto - ir - inss - sindicato); 
}
